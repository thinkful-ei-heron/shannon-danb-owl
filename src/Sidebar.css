.navToolsList {
  list-style:none;
  display: flex;
}

.navToolsList li {
  margin-left: 5px;
  margin-right: 5px;
}

.js-Sidebar {
  width: 30%;
  display: flex;
  flex-direction: column;
}