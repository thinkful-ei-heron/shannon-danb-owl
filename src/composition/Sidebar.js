import React from 'react';
import './Sidebar.css';
import Userlist from './Userlist';

function Sidebar () {
  return (
    <div className= "js-Sidebar">
      <nav>
        <ul className= "navToolsList">
          <li>Chat</li>
          <li>Participants</li>
          <li>Settings</li>
          <li>URL</li>
          <li>FAQ</li>
        </ul>
      </nav>
      <div className='Chat'>
      </div>
      <div className='Participants'>
        <Userlist />
      </div>
    </div>
  );
}


export default Sidebar;

