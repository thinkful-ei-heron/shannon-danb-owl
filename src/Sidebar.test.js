import React from 'react';
import ReactDOM from 'react-dom';
import Sidebar from './composition/Sidebar';
import renderer from 'react-test-renderer';


describe ('rendering Sidebar', () => {
  it ('renders without crashing', () => {
    const div= document.createElement('div');
    ReactDOM.render(<Sidebar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

it ('renders correctly', () => {
  const tree= renderer
  .create(<Sidebar />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
})