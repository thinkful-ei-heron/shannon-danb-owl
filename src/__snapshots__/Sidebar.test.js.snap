// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering Sidebar renders correctly 1`] = `
<div
  className="js-Sidebar"
>
  <nav>
    <ul
      className="navToolsList"
    >
      <li>
        Chat
      </li>
      <li>
        Participants
      </li>
      <li>
        Settings
      </li>
      <li>
        URL
      </li>
      <li>
        FAQ
      </li>
    </ul>
  </nav>
  <div
    className="Chat"
  />
  <div
    className="Participants"
  >
    <div>
      <div
        className="eachParticipantInfo"
      >
        <img
          alt="avatar"
          className="avatar"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
        <ul
          className="participantInfo"
        >
          <li>
            Koren Templeton
          </li>
          <li>
            <span
              className="greenCircle"
            />
            on stage
          </li>
        </ul>
      </div>
      <div
        className="eachParticipantInfo"
      >
        <img
          alt="avatar"
          className="avatar"
          src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
        />
        <ul
          className="participantInfo"
        >
          <li>
            Caty Flucker
          </li>
          <li>
            <span
              className="greenCircle"
            />
            in session
          </li>
        </ul>
      </div>
      <div
        className="eachParticipantInfo"
      >
        <img
          alt="avatar"
          className="avatar"
          src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
        />
        <ul
          className="participantInfo"
        >
          <li>
            Axe Kubicka
          </li>
          <li>
            <span
              className="greyCircle"
            />
            left session
          </li>
        </ul>
      </div>
      <div
        className="eachParticipantInfo"
      >
        <img
          alt="avatar"
          className="avatar"
          src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
        />
        <ul
          className="participantInfo"
        >
          <li>
            Frank Runciman
          </li>
          <li>
            <span
              className="greenCircle"
            />
            in session
          </li>
        </ul>
      </div>
      <div
        className="eachParticipantInfo"
      >
        <img
          alt="avatar"
          className="avatar"
          src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
        />
        <ul
          className="participantInfo"
        >
          <li>
            Ashla Attwool
          </li>
          <li>
            <span
              className="greenCircle"
            />
            on stage
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
